docs/README.md

```diff
@@ -0,0 +1,9 @@
+# Style Guides
+
+This repository is used to store localization style guides for Mozilla communities. All style guides and related documents are listed in the menu on the left of the page.
+
+## Updating the documentation
+
+We use a [Github repository](https://github.com/mozilla-l10n/styleguides/) to store all documents. Simply open a pull request adding the new file or updating an existing document. Make sure to follow [these style guidelines](https://github.com/mozilla-l10n/documentation/blob/master/misc/documentation_styleguide.md), and note that all pull requests need to be reviewed before merging. The instruction for creating  new community style guides are part of the [guidelines for community style guides](guidelines/styleguide_guidelines.md).
```

Comment:
Github -> GitHub
instruction for -> instructions for
also remove double space

Reply:
Thank you, fixed now.